File: test/fixtures/ruby/j_patch_array.rb
require 'net/http'

uri = URI('http://localhost:28139/patch')
req = Net::HTTP::Patch.new(uri)
req.content_type = 'application/x-www-form-urlencoded'

req.body = 'item[]=1&item[]=2&item[]=3'

req_options = {
  use_ssl: uri.scheme == 'https'
}
res = Net::HTTP.start(uri.hostname, uri.port, req_options) do |http|
  http.request(req)
end

File: test/fixtures/r-httr2/j_patch_array.r
library(httr2)

request("http://localhost:28139/patch") |>
  req_method("PATCH") |>
  req_body_raw(
    "item[]=1&item[]=2&item[]=3",
    type = "application/x-www-form-urlencoded"
  ) |>
  req_perform()
